src/System.Management/tests/WmiTestHelper.cs

```diff
@@ -10,7 +10,11 @@ namespace System.Management.Tests
     public static class WmiTestHelper
     {
         private static readonly bool s_isElevated = AdminHelpers.IsProcessElevated();
-        private static readonly bool s_isWmiSupported = PlatformDetection.IsWindows && PlatformDetection.IsNotWindowsNanoServer && !PlatformDetection.IsUap;
+        private static readonly bool s_isWmiSupported =
+                                            PlatformDetection.IsWindows &&
+                                            PlatformDetection.IsNotWindowsNanoServer &&
+                                            PlatformDetection.IsNotWindowsIoTCore &&
+                                            !PlatformDetection.IsUap;
```

Comment:
Curious why we have "IsNot" for some properties but then use "!" for others?

Reply:
Existing inconsistency?

Reply:
:)